.github/workflows/cypress.yaml

```
@@ -0,0 +1,148 @@
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+
+jobs:
+  Install:
+    runs-on: ubuntu-latest
+    env:
+        REACT_APP_TALAWA_URL: http://localhost:4000/graphql
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 8

(indentation)